example/lib/control_accent_color.dart

```diff
@@ -5,7 +5,7 @@ import 'package:flutter/material.dart';
 class ControlAccentColorExample extends StatelessWidget {
```

Comment:
Please rename this demo's class and file to remove the word "control"

Reply:
Done